gaurav/src/App.tsx

```diff
@@ -1,9 +1,52 @@
-import React from "react";
 import { MarkTime } from "./components/MarkTime";
+import { BrowserRouter, Link, Route, Router, Routes } from "react-router-dom";
+import { Calendar, Clock, House } from "lucide-react";
+import { Root } from "./routes/Root";
+import { Log } from "./components/Log";
 function App() {
-  return <div className="font-sans flex justify-center items-center">
-    <MarkTime />
-  </div>;
+  return (
+    <>
+      <BrowserRouter>
+        <div className="flex justify-center items-center fixed bottom-0 w-full border">
+          <ul className="flex justify-between md:justify-center items-center bg-white gap-6 p-2 px-8 w-full">
+            <MenuLink name="Home" route="/" iconName={<House className="w-5 h-5 text-gray-500" />} />
+            <MenuLink name="Log" route="login" iconName={<Calendar className="w-5 h-5 text-gray-500" />} />
+            <MenuLink name="Timelog" route="timelog" iconName={<Clock className="w-5 h-5 text-gray-500" />} />
+          </ul>
```

Comment:
You can reuse the className by declaring a variable as it is being repeated here for icons.